apps/codeforafrica/src/lib/api.netlify-cms/getProjects.js

```diff
@@ -0,0 +1,58 @@
+import { join } from "path";
+
+import { marked } from "marked";
+
+import getBadges from "./getBadges";
+import getDonors from "./getDonors";
+import getPartners from "./getPartners";
+import { getCollectionSlugs, getCollectionBySlug } from "./utils";
+
+const projectsDir = join(process.cwd(), "content/projects");
+
+export default function getProjects(fields = []) {
+  const slugs = getCollectionSlugs(projectsDir);
+  const projects = [];
+  slugs.forEach((_slug) => {
+    const collection = getCollectionBySlug(projectsDir, _slug, fields);
+    const project = collection.items;
+    if (fields.includes("badges")) {
+      const badges = getBadges(["id", "name", "content", "date"]);
+      project.badges = badges?.filter((badge) =>
+        project.badges.id.includes(badge.id)
```

Comment:
Why do we have `badges.id` and not just badges? (similar to just `donors` or `projects`).